src/com/studentmanagementsystem/presentation/Main.java

```diff
@@ -0,0 +1,85 @@
+package com.studentmanagementsystem.presentation;
+
+import com.studentmanagementsystem.data.list.*;
+import com.studentmanagementsystem.logic.*;
+import com.studentmanagementsystem.presentation.*;
+
+public class Main {
+
+    public static void main(String[] args)
+        {
+            final int ADD_STUDENT_MENU_ITEM = 1;
```

Comment:
Maybe it is better to move this to your menu class, because it belongs to the menu, and at the switch cases you only need "case Menu.ADD_STUDENT_MENU_ITEM: { ... }"